ecma402/README.md

```diff
@@ -0,0 +1,50 @@
+# [ECMAScript Internationalization API Specification](https://github.com/tc39/ecma402) proposals   
+
+ - [Stage 0 Proposals](stage-0-proposals.md)
+ - [Finished Proposals](finished-proposals.md)
+ 
+ [ECMAScript](../README.md) proposals
+
+## Active proposals
+
+Proposals follow [this process document](https://tc39.github.io/process-document/).
+This list contains only stage 1 proposals and higher that have not yet been withdrawn/rejected, or become finished.
+
+| 🚀 | Proposal                                                         | Champion                                        | Stage |
+|---|------------------------------------------------------------------|-------------------------------------------------| ------|
+|   | [Intl.Segmenter: Unicode segmentation in JavaScript][]           | Daniel Ehrenberg                                | 3 |
+|   | [Intl.RelativeTimeFormat][]                                      | Caridy Patiño, Eric Ferraiuolo                  | 2 |
+|   | [Intl.ListFormat][]                                              | Zibi Braniecki                                  | 2 |
+|   | [Exposing Abstract Operations & Locale Info][]                   | Zibi Braniecki                                  | 2 |
+|   | [Intl.Locale][]                                                  | Zibi Braniecki, Daniel Ehrenberg                | 1 |
+|   | [Intl.DurationFormat][]                                          | Zibi Braniecki                                  | 1 |
+|   | [Intl.UnitFormat][]                                              | Zibi Braniecki                                  | 1 |
+|   | [DateTimeFormat dateStyle & timeStyle][]                         | Zibi Braniecki                                  | 1 |
+
+🚀 means the champion thinks it's ready to advance but has not yet presented to the committee.
+
+### Contributing new proposals
+
+Please see [Contributing to ECMAScript](/CONTRIBUTING.md) for the most up-to-date information on contributing proposals to this standard.
+
+### Onboarding existing proposals
+
+Proposals that are Stage 1 and above must be transferred to [the TC39 GitHub organization](https://github.com/tc39) for discoverability and archival purposes. To onboard a proposal that lives outside the TC39 organization:
+
+1. Transfer your repository to the [@tc39-transfer](http://github.com/tc39-transfer) organization
+  - if you are a TC39 delegate, but not an admin in that organization, please contact @LJHarb
+2. @bterlson will transfer your repository to the TC39 organization the next chance he gets.
+
+Note that as part of the onboarding process your repository name may be normalized. Don't worry, redirects will continue to work.
```

Comment:
This isn't really true; GitHub pages just breaks. However, this is the process anyway.

Reply:
Yeah true; this (which was copypasted from the main readme) should be clarified to say that repo URLs will redirect and github pages will break.